### PR TITLE
docs(component): add missing `async` pipe and change `0` to falsy

### PR DESCRIPTION
**Pull request by meeroslav**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/component/let.md [hunk 1]
```diff
@@ -5,18 +5,18 @@ It also helps with several internal processing under the hood.
...
   </app-number-special>
 </ng-container>
  ```
 
-The problem is `*ngIf` is also interfering with rendering and in case of a `0` the component would be hidden
+The problem is `*ngIf` is also interfering with rendering and in case of a falsy value the component would be hidden
 
 The `*ngrxLet` directive take over several things and makes it more convenient and save to work with streams in the template
-`<ng-container *let="observableNumber$ as c"></ng-container>`
+`<ng-container *ngrxLet="observableNumber$ as c"></ng-container>`
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/component/let.md]: Thanks for spotting this!
I did a find, and also saw `*let` was used in the TS docs, do you mind changing that too please?

https://github.com/ngrx/platform/blob/master/modules/component/src/let/let.directive.ts#L80

**Comment by BioPhoton** [comment]
Thanks for spotting the wrong docs!!

File: projects/ngrx.io/content/guide/component/let.md (hunk 1 above)
Comment (meeroslav) [reply projects/ngrx.io/content/guide/component/let.md:19]: Done! Thank you for pointing it out.

**Comment by timdeschryver** [comment]
Thanks @meeroslav !

**Comment by meeroslav** [comment]
Thank you @timdeschryver for a quick check. And of course, thanks to @BioPhoton for the feature.

**Pull request merged**